### ISSUE TITLE
sst-info does not work after external core/elements split

### ISSUE THREAD

**Issue by allevin** [sstsimulator/sst-core#78]
sst-info does not find the registered external elements

**Comment by bjmoor** [comment]
Here's a work-around fix, until this is fixed properly:

```
diff --git a/src/sst/core/sstinfo.cc b/src/sst/core/sstinfo.cc
index be6d06b..0b84c1c 100644
--- a/src/sst/core/sstinfo.cc
+++ b/src/sst/core/sstinfo.cc
@@ -39,7 +39,7 @@ using namespace SST;
 
 // Global Variables
 ElemLoader*                              g_loader;
-std::string                              g_searchPath = SST_ELEMLIB_DIR;
+std::string                              g_searchPath = SST_INSTALL_PREFIX "/lib/sst-elements-library";
 int                                      g_fileProcessedCount;
 std::vector<SSTInfoElement_LibraryInfo*> g_libInfoArray;
 SSTInfoConfig                            g_configuration;
```

**Comment by allevin** [comment]
Fixed the problem in Pull Request #106.
